Signal/src/ViewControllers/ConversationView/MessagesViewController.m

```diff
@@ -1481,6 +1492,9 @@ - (void)didPressSendButton:(UIButton *)button
         [self clearDraft];
         [self finishSendingMessage];
         [((OWSMessagesToolbarContentView *)self.inputToolbar.contentView)ensureSubviews];
+        if (didAddToProfileWhitelist) {
+            [self ensureDynamicInteractions];
```

Comment:
If we send a message in a new (empty) contact thread, we've just created it and we can auto-add this user to the profile whitelist.